cmd/completion.go

```diff
@@ -26,14 +26,15 @@ var bashCompletionCmd = &cobra.Command{
 	Use:   "completion",
 	Short: "Generates bash tab completions",
 	Long: `Running the completion command:
```

Comment:
How about this:
```
	Long: `Outputs a bash completion script for appsody to stdout.

Bash completion is optionally available for your convenience. It helps you fill out appsody commands when you type the [TAB] key. 

To Install on macOS:
	1. brew install bash-completion
	2. Make sure to update your ~/.bash_profile as instructed
	3. appsody completion > /usr/local/etc/bash_completion.d/appsody

To install on linux:
	1. ??? do you have specific install instructions for linux?`
```

Reply:
It would be nice if the markdown generator would highlight the commands, like using back-ticks in markdown, but I'm not sure how to do that, especially since this code is already surrounded by back-ticks.